### DISCUSSION TITLE
Portuguese (Brazil) translation

### DISCUSSION THREAD

**Discussion by hardBSDk** [markqvist/Reticulum#130] [Help & Questions] [answered]
I wish to translate Reticulum/related-projects website/docs to Portuguese, what I need to do?

**Comment by markqvist** [comment]
That is absolutely incredible, and a massive piece of work! Do you mean, like *all of it*, the manual, software, website, the whole thing? Or specific parts?

**Reply by markqvist** [reply]
Like, sorry, that is actually a stupid question. Let's instead focus on how to get this rolling. Where would you like to start? What do you think would provide the most value to people speaking portuguese? Any kind of usecases of the ecosystem that you see as mostly valuable in portuguese-speaking communities? We can concentrate the efforts in that direction then.

**Reply by hardBSDk** [reply]
I think the [Reticulum website](https://reticulum.network/) is the "welcome screen" for other projects of your umbrella.

I can translate everything in some days, the most important parts first (introduction, setup process and other things less important).

Thanks for this wonderful project, it's like gold lost in the depths of ocean.

I would like a real-time chat with you to ask quick questions, I can chat trough Telegram, Matrix, Discord, XMPP.

**Reply by markqvist** [reply]
Allright, that would be awesome. The website is currently made via a (very simple) custom build system, that takes sources in markdown format, and translates them to static HTML pages.

All the code and page sources is here:https://github.com/markqvist/reticulum_website

For this to work, I need to add some small functions to the build script, so it can support switching between languages.

But if you already want to get started, you can just fork the repository. I would suggest creating a copy of each source file with the ending "_pt" added to the name (that is the right language code for portuguese, right? Correct me if it is not correct for Brazilian portuguese!). So for example, "index.md" becomes "index_pt.md". This way, I can easily modify the build script to generate separate folders for the different languages.

**Reply by hardBSDk** [reply]
Thanks, I will do that.

Portuguese (Brazil) is pt-br.

**Reply by markqvist** [reply]
Sorry for being ignorant about your language, but it is a big enough difference from "Portuguese Portuguese", that we should be able to have separate versions, right? In that case I think we should use filenames like "index_pt-br.md".

**Reply by hardBSDk** [reply]
No problem xd, Portuguese (Brazil) translation is dominant when compared to Portuguese (Portugal) translations, it varies from project to project.

Both ways are Ok (_pt/_pt-br), I will use _pt-br

When implementing language option on the page, you can implement language detection based on browser or IP.

**Reply by markqvist** [reply]
Ok, you are the one who knows the stuff here, so you make the decision! I will just make sure the build tool handles it afterwards :)

**Reply by hardBSDk** [reply]
@markqvist The first part is done.

https://github.com/markqvist/reticulum_website/pull/2

I will translate the manual later.

**Reply by markqvist** [reply]
Awesome! It is merged and waiting on https://github.com/markqvist/reticulum_website/issues/3 for deployment. I will do that as soon as I can.

**Comment by markqvist** [comment, marked as answer]
The new multilingual website is now online, with your Portuguese translation as the first language other than English. Thank you very much for contributing the translation!

@hardBSDk, Can you please take a look and let me know if I messed anything up in the menu names?

**Reply by hardBSDk** [reply]
Everything is perfect, I will fix some errors that I saw.

**Reply by hardBSDk** [reply]
Fixed.

https://github.com/markqvist/reticulum_website/pull/5

**Reply by markqvist** [reply]
Thank you! I deployed the fixes to the live site.

**Comment by fab1an2** [comment]
meybe use https://pootle.translatehouse.org/ or other free systems to translate nomad and sideband too?
